miso-web/src/main/webapp/scripts/warning.js

```diff
@@ -66,9 +66,23 @@ var Warning = (function($) {
             html += '</a>';
           }
         }
-        getWarnings(target, full, 'tableMessage').forEach(function(warning) {
-          html += ' <span class="message-' + (warning.level || 'error') + '"><strong>' + warning.tableMessage + '</strong></span>';
-        });
+        var warnings = getWarnings(target, full, 'tableMessage');
+        if (warnings && warnings.length) {
+          var level = 'info';
+          if (warnings.some(function(warning) {
+            return !warning.level || warning.level === 'error';
+          })) {
+            level = 'error';
+          } else if (warnings.some(function(warning) {
+            return !warning.level || warning.level === 'important';
+          })) {
+            level = 'important';
+          }
+          html += ' <div class="tooltip"><span class="message-' + level + '">⚠</span><span class="tooltiptext">'
+              + warnings.map(function(warning) {
+                return warning.tableMessage;
+              }).join(';<br>') + '</span></div>';
```

Comment:
This looks as though it draws 1 '⚠' for all the warnings, should we consider having 1 '⚠' per warning instead?

Reply:
I don't think additional icons will be useful as they already use the colour of the most severe warning, and the point of this change is to save space. All full warning messages are included in the tooltip